src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -208,6 +206,14 @@ export class GuidedTourService {
         this.resetTour();
     }
 
+    public checkTourStateFinished(guidedTour: GuidedTour): boolean {
```

Comment:
Please add documentation

Reply:
My bad, I missed this method. I added the documentation :)